src/app/components/LiveLabel/index.tsx

```diff
@@ -50,15 +51,23 @@ const LiveLabel = ({
   }
 
   const circle = [
-    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+    [styles.pulseContainer, className === 'first-promo' && styles.firstPromo],
   ];
```

Comment:
I've just realised (although this seems to work OK) - are there too many [] here? I think the css property can accept a single css object or an array of them. Technically this is a nested array. 

What if we had the following:

```suggestion
const circle = [
    styles.pulseContainer, 
    className === 'first-promo' && styles.firstPromo,
  ];
```